client/src/main/java/io/pravega/keycloak/client/KeycloakAuthzClient.java

```diff
@@ -56,6 +63,14 @@ public KeycloakAuthzClient(AuthzClient client, TokenCache tokenCache) {
         this.tokenCache = tokenCache;
     }
 
+    public void setHttpMaxRetries(int httpMaxRetries) {
+        this.httpMaxRetries = httpMaxRetries;
+    }
+
+    public void setHttpInitialDelayMs(int httpInitialDelayMs) {
+        this.httpInitialDelayMs = httpInitialDelayMs;
```

Comment:
This is setting a static variable from a non-static context. You might have intended `httpInitialDelayMs ` to be a class member instead